Utils/py_utils.py

```diff
@@ -1,5 +1,22 @@
 import os
```

Comment:
this functions seem importent. Are you sure they are utils?

Reply:
yes because py_utils is for holding testing utils